install/operator/pkg/syndesis/action/install.go

```diff
@@ -91,6 +91,38 @@ func (a *installAction) Execute(ctx context.Context, syndesis *v1alpha1.Syndesis
 		})
 	}
 
+	secret := &corev1.Secret{}
+	err = a.client.Get(ctx, types.NamespacedName{Namespace: syndesis.Namespace, Name: SyndesisPullSecret}, secret)
+	if err != nil && !k8serrors.IsNotFound(err) {
+		return err
+	}
+
+	// Link the builder service account to the image pull/push secret if it exists
+	if secret != nil {
+		builder := &corev1.ServiceAccount{}
+		err = a.client.Get(ctx, types.NamespacedName{Namespace: syndesis.Namespace, Name: "builder"}, builder)
```

Comment:
Is the builder in the syndesis namespace or does it exist in the openshift namespace?

Reply:
Well, I guess they don't exist in any specific namespace, right?

Reply:
There is one in each namespace by default.